components/consentRequestForm/styles.js

```diff
@@ -307,6 +313,9 @@ export default function styles(theme) {
     },
     "purposes-container": {
       padding: "2rem 0",
+      textOverflow: "ellipsis",
```

Comment:
In order for ellipsis to show up, we need to have also `white-space` set to `nowrap`.